### ISSUE TITLE
A few of the Lunar Festival coins only appear on a "single" map...

### ISSUE THREAD

**Issue by TickMan** [Vladinator89/wow-addon-candybuckets#2]
Another minor observation... almost all of the *coin* icons successfully appear on "multiple" maps (*continent, zone, city,* etc.), with the exception of:

http://www.wowhead.com/quest=8677/darkhorn-the-elder

Visible on the ***Durotar*** map, but not on the ***Kalimdor*** or ***Orgrimmar*** maps.

http://www.wowhead.com/quest=29734/deepforge-the-elder
http://www.wowhead.com/quest=29735/stonebrand-the-elder

Both visible on the ***Deepholm*** map, but not on the ***Maelstrom*** map.

Thanks again for one of my favorite addons!  <WAVE>
-TM

**P.S.**  It'd also be nice if the ***Azeroth*** map somehow indicated which continents still contained coins/candy/etc., but I'm guessing that something so minor probably isn't worth the effort...  ;^)

**Comment by Vladinator89** [comment]
I'd like you to know that I have noted these Elders and will look into it for next event.

The Azeroth map and Maelstrom map are special and are tricky to use, as to the game these aren't real maps, so translating the coordinates from the zones will be tricky. Perhaps just a overlay saying how many nodes are available would suffice, I will in any case see what I can do.

**Comment by TickMan** [comment]
> I'd like you to know that I have noted these Elders and will look into it for next event.

Nice... glad to know that this wonderful addon is still getting cared-for! 

(_...but didn't you make the same claim regarding issue_ **#1**_?_) 😜

> The Azeroth map and Maelstrom map are special and are tricky to use, as to the game these aren't real maps, so translating the coordinates from the zones will be tricky. Perhaps just a overlay saying how many nodes are available would suffice, I will in any case see what I can do.

To be honest, it wouldn't even need node coordinates/quantities... just a simple binary "_Still goodies here!_ indicator of some sort (so you'd know to click that continent for the actual icons) would suffice.
¯\\\_(ツ)_/¯

Keep being awesome!
-TM

**Comment by Vladinator89** [comment]
Yeah. I was thinking of something like that. To at least indicate on the world map, what areas one should click at and inspect.

Every year there is something missing, and every year I am too slow to fix it for said year. These events come and go so quickly.
